style.css

```diff
@@ -0,0 +1,143 @@
+:root {
+  --bg-purple: #6070ff;
+  --bg-dark-blue: #172b4d;
+  --bg-lavender: #ebebff;
+  --bg-slate-grey: #505f79;
+  --text-dark-blue: #344563;
+  --font-poppins: "Poppins", sans-serif;
+}
+
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: white;
+}
+
+html body {
+  max-width: 100%;
+  overflow-x: hidden;
+}
+
+*:not(i) {
+  font-family: var(--font-poppins);
+}
+
+.mheadline {
+  background-image: url("./images/header-shapes.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  align-items: flex-start;
+  flex-direction: column;
+  justify-content: flex-start;
+  padding: 6rem 1.5rem;
+  background-color: #fff;
+  border-bottom-left-radius: calc(10rem / 3);
+}
+
+.container {
+  min-height: 650px;
+}
+
+.after-body {
+  z-index: 1;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+h1 {
+  color: var(--text-dark-blue);
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+}
+
+h3 {
+  color: var(--bg-dark-blue);
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+}
+
+h6 {
+  color: var(--bg-purple);
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+p {
+  padding: 1rem 0;
+  color: var(--text-dark-blue);
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+}
```

Comment:
- [ ] Please 🙏 make the `font-weight` a bit lighter match the `<p>` in [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=tG1jh7hS4l814OsZ-0) as it is the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md) of the project to make it as close to figma design as possible

> Your website should look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

**_Figma design_**

<img width="289" alt="Screenshot 2023-02-17 at 10 20 33 AM" src="https://user-images.githubusercontent.com/44798044/219559090-96598344-a028-4b03-810e-46d609b71b30.png">

**_Your design_**

<img width="464" alt="Screenshot 2023-02-17 at 10 20 20 AM" src="https://user-images.githubusercontent.com/44798044/219559114-3c995628-94de-40eb-b431-ce839b65b7d3.png">